src/pushsource/_impl/source.py

```diff
@@ -6,25 +6,14 @@
 
 import pkg_resources
 
-from six.moves.urllib import parse
+from urllib import parse
 
 from pushsource._impl.helpers import wait_exist
 
 LOG = logging.getLogger("pushsource")
 
 
 def getfullargspec(x):
```

Comment:
Minor: given that this doesn't do anything any more other than just call inspect.getfullargspec, maybe this function should be removed and the callers updated to use the function from inspect?

Reply:
My mistake, I totally forgot about that one. Thanks for the reminder